### PR TITLE
[patch] Grpc ecosystem master

### PR DESCRIPTION
**Pull request by tony-assembly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,24 @@
...
-
 The `grpc_health_probe` utility allows you to query health of gRPC services that
 expose service their status through the [gRPC Health Checking Protocol][hc].
 
+`grpc_health_probe` is meant to be used for health checking gRPC applications in
+[Kubernetes][k8s], using the [exec probes][execprobe].
+
+> :warning: [**Kubernetes v1.23 has now introduced built-in gRPC health checking**][k8s-new]
+> capability as an alpha feature. As a result, you might no longer need to use this tool and use the
+> native Kubernetes feature instead.
```
Comment (awilmore) [inline README.md:14]: @tony-assembly interesting... might be worth looking into (where's bmon when you need him?)

**Review by awilmore — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (tony-assembly) [reply README.md:14]: i know 😢 how do you propose we give this a k8s feature a crack? I'd be happy to help

Comment (tony-assembly) [reply README.md:14]: looks like we might just be able to plug it in! https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe

Comment (awilmore) [reply README.md:14]: We can try it out in RND... I'll create a ticket

Comment (tony-assembly) [reply README.md:14]: 🙌

**Pull request merged**